libraries/AP_HAL_ChibiOS/hwdef/QioTekZealotF427/hwdef.dat

```diff
@@ -0,0 +1,229 @@
+# IMU Invensensev2 SPI:icm20649 ROTATION_NONE
+
+# one baro, check both addressess
+BARO DPS280 SPI:dps310
+BARO MS56XX I2C:0:0x76
+BARO MS56XX I2C:0:0x77
+
+
+# enable RAMTROM parameter storage
+define HAL_WITH_RAMTRON 1
+# enable FAT filesystem support
+define HAL_OS_FATFS_IO 1
+# now some defines for logging and terrain data files
+define HAL_BOARD_LOG_DIRECTORY "/APM/LOGS"
+define HAL_BOARD_TERRAIN_DIRECTORY "/APM/TERRAIN"
+
+# define the order that I2C buses 
+I2C_ORDER I2C2 I2C1
+# this defines the default maximum clock on I2C devices.
+define HAL_I2C_MAX_CLOCK 100000
+I2C_ORDER I2C2 I2C1
+PB8 I2C1_SCL I2C1 
+PB9 I2C1_SDA I2C1 
+PB10 I2C2_SCL I2C2
+PB11 I2C2_SDA I2C2
+
+# look for I2C compass
+COMPASS IST8310 I2C:0:0x0E false ROTATION_YAW_270
+define HAL_PROBE_EXTERNAL_I2C_COMPASSES
+
+
+# PWM out pins
+PA0 TIM2_CH1 TIM2 PWM(1) GPIO(50)
+PA1 TIM2_CH2 TIM2 PWM(2) GPIO(51)
+PA2 TIM2_CH3 TIM2 PWM(3) GPIO(52)
+PA3 TIM2_CH4 TIM2 PWM(4) GPIO(53)
+PE9 TIM1_CH1 TIM1 PWM(5) GPIO(54)
+PE11 TIM1_CH2 TIM1 PWM(6) GPIO(55)
+PE13 TIM1_CH3 TIM1 PWM(7) GPIO(56)
+PE14 TIM1_CH4 TIM1 PWM(8) GPIO(57)
+
+PD13 TIM4_CH2 TIM4 PWM(9) GPIO(58)
+PD12 TIM4_CH1 TIM4 PWM(10) GPIO(59)
+PD15 TIM4_CH4 TIM4 PWM(11) GPIO(60)
+PD14 TIM4_CH3 TIM4 PWM(12) GPIO(61)
+PC9  TIM3_CH4 TIM3 PWM(13) GPIO(62) NODMA
+PC8  TIM3_CH3 TIM3 PWM(14) GPIO(63) NODMA
+
+PC14 EXTERN_GPIO1 GPIO(64)
+PC13 EXTERN_GPIO2 GPIO(65)
+PE4  EXTERN_GPIO3 GPIO(66)
+PE6  EXTERN_GPIO4 GPIO(67)
+define RELAY1_PIN_DEFAULT 64
+define RELAY2_PIN_DEFAULT 65
+define RELAY3_PIN_DEFAULT 66
+define RELAY4_PIN_DEFAULT 67
+
+# also USART6_RX for serial RC
+PC7 TIM8_CH2 TIM8 RCIN PULLUP LOW DMA_CH0 
+
+# LED setup is similar to PixRacer
+define HAL_HAVE_PIXRACER_LED
+PE3 LED_RED OUTPUT GPIO(10)
+PE2 LED_GREEN OUTPUT GPIO(11)
+PE1 LED_BLUE OUTPUT GPIO(12)
+PE0 LED_YELOW OUTPUT GPIO(13)
+
+define HAL_GPIO_A_LED_PIN 10
+define HAL_GPIO_B_LED_PIN 11
+define HAL_GPIO_C_LED_PIN 12
+
+define HAL_GPIO_LED_ON  0
+define HAL_GPIO_LED_OFF 1
+
+# analog in
+PC0 PRESSURE_SENS ADC1 SCALE(2)
+PC1 RSSI_IN ADC1 SCALE(2)
+
+# define the primary battery connectors.
+
+PC2 BATT_CURRENT_SENS ADC1 SCALE(2)
+PC3 BATT_VOLTAGE_SENS ADC1 SCALE(2)
+PB0 BATT2_CURRENT_SENS ADC1 SCALE(2)
+PB1 BATT2_VOLTAGE_SENS ADC1 SCALE(2)
+PC4 VDD_5V_SENS ADC1 SCALE(2)
+PC5 FMU_SERVORAIL_VCC_SENS ADC1 SCALE(12)
+
+define HAL_BATT_VOLT_PIN 13
+define HAL_BATT_CURR_PIN 12
+define HAL_BATT2_VOLT_PIN 9
+define HAL_BATT2_CURR_PIN 8
+define HAL_BATT_VOLT_SCALE 10.7
+define HAL_BATT_CURR_SCALE 17
+
+define HAL_HAVE_IMU_HEATER 1
+PE12 HEATER_EN OUTPUT LOW GPIO(5)
+define HAL_HEATER_GPIO_PIN 5
+
+define HAL_HAVE_SAFETY_SWITCH 1 
+PE7 LED_SAFETY OUTPUT
+PE8 SAFETY_IN INPUT PULLDOWN
+PE5 TIM9_CH1 TIM9 ALARM
+
+# setup for OSD
+define OSD_ENABLED 1
+define HAL_OSD_TYPE_DEFAULT 1
+ROMFS_WILDCARD libraries/AP_OSD/fonts/font*.bin
+
+# drdy pins
+PA15  DRDY_ICM20689 INPUT
+PA10  DRDY_ICM20602 INPUT
+# PD11 DRDY_ICM20649 INPUT
+
+#define AP_FEATURE_BRD_PWM_COUNT_PARAM 1
```

Comment:
AP_FEATURE_BRD_PWM_COUNT_PARAM line can be removed

Reply:
OK, thanks.